promotion/suggestions.go

```diff
@@ -166,8 +171,58 @@ func (service *Service) RedeemAndCreateSuggestionEvent(ctx context.Context, cred
 		return nil
 	}
 
-	service.eventChannel <- suggestion
-	// TODO emit event
+	// kafka event producer below - this could probably put into a generic module
+	kafkaBrokers := os.Getenv("KAFKA_BROKERS_STRING")
+	kafkaWriter := kafka.NewWriter(kafka.WriterConfig{
```

Comment:
we can probably toss the writer under the service struct, per docs:
https://godoc.org/github.com/segmentio/kafka-go#NewWriter
> Instances of Writer are safe to use concurrently from multiple goroutines